x-pack/plugins/canvas/public/apps/workpad/workpad_app/workpad_app.scss

```diff
@@ -29,6 +31,7 @@
   flex-grow: 0;
   flex-basis: auto;
   padding: $euiSizeM $euiSize $euiSizeS $euiSize;
+  font-size: $canvasLayoutFontSize;
```

Comment:
Looks good! I suppose the font family is set higher in the class tree.

Reply:
That is correct, the font is set globally for Kibana.